packages/@okta/vuepress-site/docs/concepts/authentication/index.md

```diff
@@ -7,80 +7,43 @@ meta:
 
 Okta provides the OAuth 2.0 and OpenID Connect implementations, the Authentication API, the Sign-in Widget, and the Auth JS SDK:
 
 * [OAuth 2.0, OpenID Connect, and Social Login](#oauth-20-openid-connect-and-social-login)
+* [Authentication API](#authentication-api)
 * [Sign-In Widget](#sign-in-widget)
 * [Auth SDK](#auth-sdk)
-* [Authentication API](#authentication-api)
 
 ### OAuth 2.0, OpenID Connect, and Social Login
 
-Okta allows you to control access to your application using both the OAuth 2.0 and OpenID Connect specifications. You can use Okta as your authorization server to retain all of your user information, and grant users tokens to control their authorization and authentication. Okta also supports using social login and pulling user's external data into your Okta org.
+Okta allows you to control access to your application using both the OAuth 2.0 and OpenID Connect specifications. You can use Okta as your authorization server to retain all of your user information, and grant users tokens to control their authorization and authentication. Okta also supports using social logins and pulling a user's external data into your Okta org.
 
 To find out more about Okta's OAuth 2.0 and OpenID Connect implementations see [OAuth 2.0 and OpenID Connect](/docs/concepts/auth-overview/).
 
 ### Sign-In Widget
 
-The [Okta Sign-in Widget](/code/javascript/okta_sign-in_widget/)
-provides an embeddable Javascript sign-in implementation that can
-easily be embedded into your customized login page.  The Sign-in
-widget carries the same feature set in the standard Okta sign-in page
-of every tenant - with the added flexibility to change the
-look-and-feel.  Included in the widget is support for password reset,
-forgotten password and strong authentication - all of which are  driven
-by policies configured in Okta.  Developers don't have to write a
-single line of code to trigger these functions from within the widget.
-For consumer facing sites, social providers are also supported in the
-widget.
+The [Okta Sign-in Widget](/code/javascript/okta_sign-in_widget/) provides an embeddable Javascript sign-in implementation that can easily be embedded into your customized login page.  The Sign-in widget carries the same feature set in the standard Okta sign-in page of every tenant along with the added flexibility to change the look and feel.  Included in the widget is support for password reset, forgotten password and strong authentication functionality - all of which are  driven by policies configured in Okta.  Developers don't have to write a single line of code to trigger these functions from within the widget. For consumer facing sites, social providers are also supported in the widget.
```

Comment:
fixing some things here:

The [Okta Sign-in Widget](/code/javascript/okta_sign-in_widget/) provides an embeddable Javascript sign-in implementation that can easily be embedded into your customized sign-in page. The Sign-In Widget carries the same feature set in the standard Okta sign-in page of every tenant along with the added flexibility to change the look and feel. Included in the widget is support for password reset, forgotten password, and strong authentication functionality - all of which are driven by policies configured in Okta. Developers don't have to write a single line of code to trigger these functions from within the widget. For consumer facing sites, social providers are also supported in the widget.

Reply:
Fixes made